ocaml/database/database_test.ml

```diff
@@ -12,6 +12,8 @@
  * GNU Lesser General Public License for more details.
  *)
 
+let name_label = name_label
```

Comment:
Well that's odd, don't know how I managed to type that in. However, it does reveal that that file is not actually compiled at all. I'll remove it!